plugins/proxies/lib_namelib.eclmod

```diff
@@ -0,0 +1,29 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2020 HPCC Systems® .
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+############################################################################## */
+
+/* Proxy plugin definition for (EE only) name library */
+
+export NameLib := SERVICE:plugin('name'),fold
+  string    FirstNameToToken(const string20 name) : c, pure, entrypoint='nameFirstNameToToken';
+  string20  TokenToFirstName(const string token) : c, pure, entrypoint='nameTokenToFirstName';
+  unsigned4 TokenToLength(const string name) : c, pure, entrypoint='nameTokenToLength';
+END;
+export fnstring := TYPE
```

Comment:
This type could be deleted in 7.8.  I'm not convinced the plugin is used at all.